### ISSUE TITLE
October 22nd meetup

### ISSUE THREAD

**Issue by SheffieldKevin** [SheffieldSwift/Operations#4]
We have Union-St booked for this date. There is someone else who is hoping we can move (change venue or date). They would need the whole floor. Can we be flexible? or should we just keep our booking. What alternatives are there?

**Comment by lukestringer90** [comment]
Are we going to reschedule for the 22nd? Would be good to confirm this publicly here and then start posting about it on Twitter / FB etc

**Comment by SheffieldKevin** [comment]
Yes. Though I am not sure that confirming publicly until after the September meeting is sensible as it is just likely to create confusion. I have booked union st for the 22nd of October.

**Comment by lukestringer90** [comment]
Agreed - we should wait until after the September meeting.
